src/Effects/NNEDI3/NNEDI3_nns16_win8x6.hlsl

```diff
@@ -0,0 +1,948 @@
+		+ W(2, -1111041043, 1035331132, 1017605134, -1131113128)
+		+ W(3, 1026247587, -1110742584, 1047524760, -1095527502)
+		+ W(4, 1042485668, -1130744068, 1009982783, -1113918027) + W(5, 1038280501, 1041941518, -1110999603, 992723116)
+		+ W(6, -1136883881, 1032009669, -1096311074, 1051037928)
+		+ W(7, -1106204846, 1025830203, -1128223794, 1025751155)
+		+ W(8, 1042402294, -1106649743, -1132447358, 1017749654) + W(9, 999596614, -1126831290, -1118872454, 1032615945)
+		+ W(10, 1002160934, -1127230527, -1126850910, 1033490448)
+		+ W(11, 1023947050, -1111971999, 971034337, 1018668086);
+	sum2 = W(0, 988660617, 1017543700, 1015794522, -1133704409) + W(1, 1003471274, -1140119133, -1145776834, 1002138986)
+		 + W(2, 1001599498, 1024621822, -1135257421, -1136500105)
+		 + W(3, -1133422913, 1031822055, 1041494739, -1102581932)
+		 + W(4, 970658596, -1163479081, -1126488793, 1032911160) + W(5, 1056510750, -1089051586, 1026713544, 1009057465)
+		 + W(6, 999416722, 1018658069, 1023998101, -1111744235) + W(7, 945757471, 1000517690, 999055930, 1007351961)
+		 + W(8, -1138508317, 1009295285, 998080468, -1137960905) + W(9, 987033481, -1162261577, 991201876, -1140892226)
+		 + W(10, -1156050276, -1186683976, -1179419172, 999395634)
+		 + W(11, -1141702058, -1147317506, 1007988669, -1146609818);
+	WS(1064784784, -1120346387);
+	sum1 = W(0, -1150678408, 1015721531, 1049255678, -1099108228)
+		 + W(1, -1149551256, -1136953142, 1000581420, -1110077251)
+		 + W(2, 1043607805, -1107416484, 1017163947, -1140022794)
+		 + W(3, 1006062348, -1107299655, 1059242626, -1089544734)
+		 + W(4, 1023526494, -1139533474, 1015088861, -1132691862)
+		 + W(5, -1123916922, -1130977491, 1022505321, 1012221798)
+		 + W(6, -1136518116, -1148196556, -1096371932, 1057929313)
+		 + W(7, -1104456865, 1014035238, -1126533711, 1013224070)
+		 + W(8, -1100407642, 1048500643, -1111675367, 1026165050)
+		 + W(9, 1012432222, -1124886999, -1132580564, 1035479729)
+		 + W(10, -1127245287, -1136458552, -1122704190, 1014270588)
+		 + W(11, -1102354822, 1044504531, 1007459698, 1017479699);
+	sum2 = W(0, -1140771860, 1031694512, -1104948969, -1115570202)
+		 + W(1, 1040745971, -1127298441, -1125513054, -1122230843) + W(2, 993388690, 1042093481, -1111499166, 995262946)
+		 + W(3, -1131667695, 979286214, 1026183534, 1042830623) + W(4, -1119680402, 1002124441, -1131288705, 1025077104)
+		 + W(5, -1111209187, -1112764939, 982469091, -1123012516) + W(6, 978159878, -1108853537, 1041617383, 1043422569)
+		 + W(7, -1120447085, -1129740789, 1012596136, -1102087836)
+		 + W(8, 1045410736, 1034771561, -1109907689, -1125016939)
+		 + W(9, 1011933560, -1117751010, 1030126174, 1014235016)
+		 + W(10, -1127258987, 1004566649, -1121534607, -1113389694)
+		 + W(11, 1044425994, 1025820984, -1115100280, -1119639931);
+	WS(-1088649680, 1067112300);
+	return clamp(mstd0 + 5.0 * vsum / wsum * mstd1, 0.0, 1.0);
+} // nnedi3
```

Comment:
Need a line break here

Reply:
I've added line breaks both before the return and after the function.